### PR TITLE
wishlist page added

### PR DESCRIPTION
**Pull request by aamrapaliwandhre**
chrome-capture (25)

### CODE REVIEW

**Review by MonikaJhingan — APPROVED** [review]
ui & code looks good pls use rem in later projects

**Review by pratik-kumar-sahu — APPROVED** [review]
UI and code looks nice go ahead with merging

File: Pages/HomePage/homepage.css [hunk 1]
```diff
@@ -56,7 +58,7 @@
 }
 .searchIcon{
     font-size: 20px ;
```
Comment (harshkotkar5557) [inline Pages/HomePage/homepage.css:60]: avoid giving font-size in pixels  `font-size: 20px ` replace `font-size: 1.25rem ` (20px equals to 1.25rem )

**Review by harshkotkar5557 — COMMENTED** [review]
clean code

**Pull request merged**